src/herbivoresAndCarnivores.js

```diff
@@ -2,6 +2,18 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+  health = 100;
+  hidden = false;
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
+  }
```

Comment:
only instances of Herbivore should have hide method

Reply:
Ok.